pkg/console/util.go

```diff
@@ -247,7 +247,37 @@ func toCloudConfig(cfg *config.HarvesterConfig) *k3os.CloudConfig {
 	return cloudConfig
 }
 
-func doInstall(g *gocui.Gui, cloudConfig *k3os.CloudConfig) error {
+func execute(g *gocui.Gui, env []string, cmdName string) error {
+	cmd := exec.Command(cmdName)
+	cmd.Env = env
+	stderr, err := cmd.StderrPipe()
+	if err != nil {
+		return err
+	}
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		return err
+	}
+	if err := cmd.Start(); err != nil {
+		return err
+	}
+
+	scanner := bufio.NewScanner(stdout)
+	scanner.Split(bufio.ScanLines)
+	for scanner.Scan() {
+		printToInstallPanel(g, scanner.Text())
+	}
+	scanner = bufio.NewScanner(stderr)
+	scanner.Split(bufio.ScanLines)
+	for scanner.Scan() {
+		printToInstallPanel(g, scanner.Text())
+	}
+	return cmd.Wait()
+}
+
+func doInstall(g *gocui.Gui, cloudConfig *k3os.CloudConfig, webhooks RendererWebhooks) error {
+	webhooks.Handle(EventInstallStarted)
```

Comment:
Handle() may return an error but is ignored by the caller.

Reply:
Thanks, make the function return nothing since we don't want to stop the installer if there are any errors.